enterprise-beans/4.0/_index.md

```diff
@@ -7,18 +7,25 @@ summary: "Release for Jakarta EE 9"
 Jakarta Enterprise Beans defines an architecture for the development and deployment of component-based business applications.
 
 * [Jakarta Enterprise Beans 4.0 Release Record]()
-  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
+  * [Jakarta Enterprise Beans 4.0 Release Plan](https://github.com/eclipse-ee4j/ejb-api/blob/release-plan/4.0-PLAN.adoc)
 * [Jakarta Enterprise Beans 4.0 Specification Document]() (PDF)
 * [Jakarta Enterprise Beans 4.0 Specification Document]() (HTML)
 * [Jakarta Enterprise Beans 4.0 Javadoc](./apidocs)
 * [Jakarta Enterprise Beans 4.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
   * [jakarta.ejb:jakarta.ejb-api:jar:4.0.0]()
 
+The Jakarta Enterprise Beans 4.0 intended scope includes all plans detailed in the link:https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan[Jakarta EE 9 Release Plan], which in essence involves a backwards incompatible namespace change from `javax.ejb` to `jakarta.ejb`.  In addition the Jakarta Enterprise Beans 4.0 Release Plan will involve three changes:
```

Comment:
Looks like you need to correct the syntax for the link to the Release Plan.  Your current link: syntax is not rendering correctly...  This one will work:  [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan).

Reply:
Should work now.  Resolve if all looks good.